elasticsearch/src/main/java/com/yahoo/ycsb/db/ElasticSearchClient.java

```diff
@@ -90,13 +92,15 @@ public void init() throws DBException {
         .parseBoolean(props.getProperty("elasticsearch.remote", "false"));
     Boolean newdb =
         Boolean.parseBoolean(props.getProperty("elasticsearch.newdb", "false"));
-    Builder settings = settingsBuilder().put("node.local", "true")
+    Builder settings = Settings.settingsBuilder()
+        .put("node.local", "true")
```

Comment:
We're trying to keep datastore directions defaulted to working with a remote datastore.

Instructions in the readme for bencarking with a local node is fine,  but this change would xuse it to only work that way, right?